core/src/main/java/nl/nn/adapterframework/configuration/IbisContext.java

```diff
@@ -343,15 +343,15 @@ private void digestClassLoaderConfiguration(ClassLoader classLoader,
 				ConfigurationWarnings.getInstance().setActiveConfiguration(configuration);
 
 				if(AppConstants.getInstance(classLoader).getBoolean("jdbc.migrator.active", false)) {
+					Migrator databaseMigrator = getBean("jdbcMigrator", Migrator.class);
```

Comment:
Might be best to leave this in the try/catch, as otherwise it can prevent the configuration from starting up.